capa/features/extractors/ida/helpers.py

```diff
@@ -167,6 +167,9 @@ def basic_block_size(bb):
 def read_bytes_at(ea, count):
     """ """
     segm_end = idc.get_segm_end(ea)
+    if segm_end == idaapi.BADADDR:
+        # address not in any segment
+        return
```

Comment:
lets either raise and exception or return an empty bytes here. as is, it returns `None`, which differs from the other type returned by this function.